DESCRIPTION

```diff
@@ -25,4 +25,4 @@ Suggests:
 Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 6.0.1
+RoxygenNote: 6.0.1.9000
```

Comment:
Can you please rebuild with CRAN roxygen? (You will probably need to merge recent changes from master to avoid spurious diffs)

Reply:
Upgraded to 6.1.1